plugin-parser/mlops-plugin-parser.cc

```diff
@@ -238,6 +216,73 @@ _parse_json (const gchar *json_path, mlsvc_json_type_e json_type, const gchar *a
 /**
  * @brief Handle INSTALL process. Tizen app-installer invoke this function.
  */
+static void
```

Comment:
I think Line 216~218 is for Tizen app-installer using the Tizen appfw below.
https://review.tizen.org/git/?p=platform/core/appfw/pkgmgr-info.git;a=summary

How about changing comment liked this?

```
/**
 * @brief Get config from json to app_info
 */
```

Reply:
Thanks reviews.
I'm going to add the example about the json unit test. 
and i fixed @brief.